addons/web/static/src/views/fields/properties/property_value.scss

```diff
@@ -84,4 +84,8 @@ input.o_field_property_input[type=number] {
             display: block;
         }
     }
+
+    select:not(:hover) {
```

Comment:
What is this solving, out of curiosity?

Reply:
this is removing the "triangle" of the select element if we don't hover it